src/FileStream.cpp

```diff
@@ -560,7 +565,12 @@ static bool BaseMap_Open(TFileStream * pStream, LPCTSTR szFileName, DWORD dwStre
         // Get the file size
         if(fstat64(handle, &fileinfo) != -1)
         {
+#if !defined(PLATFORM_AMIGA)

```

Comment:
Please, don't replace memory mapping with memory allocation. It's OK to just leave it unsupported if the target platform does not support memory mapped files.

Reply:
are you referring to the switch platform?

Reply:
There is
```
#if !defined(PLATFORM_AMIGA)
#if defined(PLATFORM_SWITCH)
```
If memory mapping is supported under a platform, leave `mmap` there. If not, just leave unimplemented.

Reply:
are both wrong?

Reply:
Yes. `mmap` is a function that maps file data to a memory. You get a pointer to memory that contains plain file data. `malloc` is not a replacement of it, by no means.